components/gradle.properties

```
@@ -9,7 +9,7 @@ android.useAndroidX=true
 #Versions
 kotlin.version=2.1.0
 agp.version=8.2.2
-compose.version=1.8.0-alpha01
+compose.version=1.8.0-alpha03
```

```suggestion
compose.version=1.8.0+dev2134
```
from https://github.com/JetBrains/compose-multiplatform/pull/5219#issuecomment-2677321021

> Now there is only failed test on GitHub CI due to MissingResourceException, which is because CI does not use latest version of resources component library, it has changes which are required for the test to succeed.

If you need a new functionality from Compose, please update the version to the latest dev, it is fine to use it (`1.8.0+dev2134` at the moment)

---

@igordmn thanks, but this PR requires changes that are in this same PR. How can I use a dev version that hasn't been released yet?

I think it would be best to make a new PR after this one is merged, which updates the dev version to the latest. Any other ideas?

---

On the TC all modules in the build pipeline are synchronized and use the common version.
This is GH actions problem.

---

> this PR requires changes that are in this same PR. How can I use a dev version that hasn't been released yet?

Right. I thought you need a new functionality from Compose itself.

If you need a new functionality from the Resource library, the only way is to make 2 PRs.

This issue is known and very inconvenient, we discussed that we should do something with it in the future.

---

I can create a new PR to update the compose version after this one is merged. That should fix GitHub CI.

---

@igordmn Ok, I understand. The failing tests are now ignored, and after this PR is merged I will immediately open a new PR to update compose to the latest dev and remove the Ignore annotations.

---

Also, add:
```
## Release Notes
N/A
```

---

@igordmn only the windows test is failing, all other tests are now green.

---

Thanks!

---

New PR: https://github.com/JetBrains/compose-multiplatform/pull/5260